test/agent_test.go

```diff
@@ -97,10 +109,14 @@ func TestAgents(t *testing.T) {
 			return "room job must be assigned 1 time"
 		}
 
-		if ac3.participantJobs.Load()+ac4.participantJobs.Load() != 2 {
+		if ac3.publisherJobs.Load()+ac4.publisherJobs.Load() != 2 {
```

Comment:
Assuming my understanding is correct (i. e. publisher is assigned to participant job type also), should this check for `ac3.participantJobs.Load() +...` also?

Reply:
the workers ac3 and ac4 are only registered to handle publisher jobs